src/libs/filters/rating_legacy.c

```diff
@@ -209,7 +210,10 @@ static void _rating_legacy_widget_init(dt_lib_filtering_rule_t *rule, const dt_c
   DT_BAUHAUS_COMBOBOX_NEW_FULL(rating_legacy->stars, self, NULL, N_("ratings"), _("filter by images rating"), 0,
                                _rating_legacy_changed, rating_legacy, N_("all"), N_("unstarred only"), "★", "★ ★",
                                "★ ★ ★", "★ ★ ★ ★", "★ ★ ★ ★ ★", N_("rejected only"), N_("all except rejected"));
-  dt_bauhaus_widget_set_label(rating_legacy->stars, NULL, NULL);
+  DT_BAUHAUS_WIDGET(rating_legacy->stars)->show_label = FALSE;
+  // we increase the left padding of the 5 star entry to be sure it's visible with the comparator on top
+  // we do that here to not cause trouble with shortcuts
+  dt_bauhaus_combobox_set_entry_label(rating_legacy->stars, 6, "             ★ ★ ★ ★ ★");
```

Comment:
I would remove 2 spaces here. On your screenshot, seems to be a little too much space